collections/_article/release-candidate-godot-4-0-1-rc-1.md

```diff
@@ -0,0 +1,130 @@
+
+For the time being we are primarily focusing on changes that address critical issues (such as crashes and regressions), as well as small self-contained fixes to various parts of the engine and documentation improvements. This allows us to publish this patch release quickly and with a lot of confidence that it doesn't introduce new problems and regressions in the process. As such, we are starting directly with a [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate).
+
+Please give it a try if you can. It should be safe to migrate your existing projects to 4.0.1, but to make sure of that we need your help testing the changes. If there are no significant regressions reported with release candidates, a stable version is going to be published soon. Don't forget to always make backups when moving versions, even minor. Better yet, prefer using a version control system, such as Git, and commit a version of your project before the migration.
+
+## Highlights
+
+This release candidate includes some changes which may interest a lot of users:
+
+- Multiple conditions for crashes and freezes have been addressed throughout the editor and the engine.
+
+- The project converter has been improved with new renames and corrections to its behavior around various scripting statements.
+
+- Numerous improvements to the documentation and translations.
+
+[Jump to the **Downloads** section.](#downloads)
+
+As usual, you can try it live with the [**online version of the Godot editor**](https://editor.godotengine.org/releases/4.0.1.rc1/) updated for this release.
+
+-----
+
+*The illustration picture is from* [**Abandoned Spaceship**](https://www.youtube.com/watch?v=Fm9a6FGBWbs) *by [Jaanus Jaggo (Perfoon)](https://twitter.com/JaanusJaggo), a beautiful demo scene made with Godot 4. Download and star this project on [GitHub](https://github.com/perfoon/Abandoned-Spaceship-Godot-Demo) or get it from the [Asset Library](https://godotengine.org/asset-library/asset/1733). Also follow Jaanus on [Twitter](https://twitter.com/JaanusJaggo) to see more of their work and maybe catch a glimpse of the next update to* [**BLASTRONAUT**](https://store.steampowered.com/app/1392650/BLASTRONAUT/) *!*
+
+## What's new
+
+See the full changelog [on GitHub](https://github.com/godotengine/godot/compare/4.0-stable...fc7adaab7b3856a7831d402ea2bbb27efe7b7d8a), or in a text form (sorted [by authors](https://downloads.tuxfamily.org/godotengine/4.0.1/rc1/Godot_v4.0.1-rc1_changelog_authors.txt) or [chronologically](https://downloads.tuxfamily.org/godotengine/4.0.1/rc1/Godot_v4.0.1-rc1_changelog_chrono.txt)) for an overview of all changes since 4.0-stable (102 commits – excluding merge commits ― from 49 contributors).
```

Comment:
```suggestion
See the full changelog [on GitHub](https://github.com/godotengine/godot/compare/4.0-stable...d23922ffebe48f29126c003411495737d07e5a9f) for an overview of all changes since 4.0-stable (103 commits – excluding merge commits ― from 49 contributors).
```